packages/botonic-plugin-ner/babel.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
try removing it. I'm not sure it's necesary for TS, and we have already one at the packages folder